src/Controller/InvoiceController.php

```diff
@@ -170,6 +171,28 @@ public function edit(Request $request, Invoice $invoice, InvoiceRepository $invo
         ]);
     }
 
+    #[Route('/{id}/generate-description', name: 'app_invoices_generate_description', methods: ['GET'])]
+    public function generateDescription(Invoice $invoice, $defaultInvoiceDescriptionTemplate): JsonResponse
+    {
+        $description = $defaultInvoiceDescriptionTemplate;
+
+        // Default description.
+        if (!empty($invoice->getClient())) {
+            $projectLeadName = $invoice->getClient()?->getProjectLeadName() ?? null;
+            $projectLeadMail = $invoice->getClient()?->getProjectLeadMail() ?? null;
+
+            if ($projectLeadName) {
+                $description = str_replace('%name%', $projectLeadName, $description);
+            }
+
+            if ($projectLeadMail) {
+                $description = str_replace('%email%', $projectLeadMail, $description);
+            }
+        }
+
+        return new JsonResponse(['description' => $description]);
```

Comment:
`$description` may equal `"Spørgsmål vedrørende fakturaen rettes til %name%, %email%."` here, so it may be a better idea to reply with `{"description": null}` if name and email cannot be found.